coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/person/ProposalDevelopmentPersonnelController.java

```diff
@@ -280,6 +281,12 @@ public ModelAndView certifyAnswers(@ModelAttribute("KualiForm") ProposalDevelopm
         return super.save(form);
     }
 
+    @Transactional @RequestMapping(value = "/proposalDevelopment", params = "methodToCall=retrieveCollectionPage")
+    public ModelAndView retrieveCollectionPage(@ModelAttribute("KualiForm") DocumentFormBase form, BindingResult result, HttpServletRequest request,
```

Comment:
This one too is not required. I did not notice that there is already a retrieveCollectionPage in the ProposalDevelopmentCoreController and ProposalBudgetCommonController which are being used. So this was duplicate code. Thanks for noticing.